kubernetes/aks/system/ori/base/postgresql/postgresql.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: postgresql.cnpg.io/v1
+kind: Cluster
+metadata:
+  name: ori-postgresql
+spec:
+  instances: 3
+  imageName: ghcr.io/cloudnative-pg/postgresql:15.4
```

Comment:
are we planning to have a single deployment of postgresql hosting all our different apps? I'd imagine we'd need to bring this up to the top-level and reference the existing cluster from services depending on it.

Reply:
As far as client apps we could have a single server but we would have to discuss this. For Ori I would like to separate concerns since it handles sensitive information and configuration.

Reply:
a single database server can have multiple databases which can in turn have multiple schemas and tables/views

Reply:
Yes, This is the case with the posgresql server in the folder 'postgresql'. It will be used by all hosted services from Ori. What I am saying is this a system deployment and not a client app deployment.It might complicate access control management and introduces increases risks of data breaches like authentication data.